src/components/super-spawner.js

```diff
@@ -68,8 +68,12 @@ AFRAME.registerComponent("super-spawner", {
   _emitEvents: function(entity) {
     const data = this.entities.get(entity);
     if (data.componentInitialized && data.bodyLoaded) {
-      data.hand.emit("action_grab", { targetEntity: entity });
-      entity.emit("grab-start", { hand: data.hand });
+      for (let i = 0; i < this.data.events.length; i++) {
```

Comment:
`grab-start` is for the `grabbable` component, `action_primary_down` and `action_grab` are for the `super-hands` component. The ones on the physical hands listen for a different event than the one of the cursor.

Reply:
Shouldn't `action_primary_grab` and `action_grab` both result in the `super-hand` emitting a `grab-start`?

Reply:
Assuming you relied on that, and fired the `grab-end` event to properly "reset" the hand, I believe the result of that would always be the spawner being grabbed again, because to the hand, it will always be the first object it intersected with. Notice the `targetEnity` being passed in- this explicitly tells the hand what the object it is grabbing now is.